### PR TITLE
Form Adapter

### PR DESCRIPTION
**Pull request by auslin-aot**
# Issue Tracking

JIRA: 
Issue Type: FEATURE  :sparkles:
https://aottech.atlassian.net/browse/FWF-1446

### CODE REVIEW

**Review by AbijithS-aot — APPROVED** [review]

File: deployment/docker/sample.env [hunk 1]
```diff
@@ -172,6 +172,10 @@ WEBSOCKET_SECURITY_ORIGIN=http://{your-ip-address}:3000
 #bpmn settings
 BPM_BASE_URL=http://{your-ip-address}:8000/engine-bpm
 
+#custom submission
+CUSTOM_SUBMISSION_ENABLED=false
+CUSTOM_SUBMISSION_URL=custom_url
```
Comment (shibin-aot) [inline deployment/docker/sample.env:177]: give a sample url if possible to help user configure it

File: forms-flow-bpm/sample.env [hunk 2]
```diff
@@ -67,4 +67,8 @@ WEBSOCKET_SECURITY_ORIGIN=http://{your-ip-address}:3000
 DATA_ANALYSIS_URL=http://{your-ip-address}:6000/analysis
 
 #bpmn settings
-BPM_BASE_URL=http://{your-ip-address}:8000/engine-bpm
\ No newline at end of file
+BPM_BASE_URL=http://{your-ip-address}:8000/engine-bpm
+
+#custom submission settings
+CUSTOM_SUBMISSION_URL=custom_url
```
Comment (shibin-aot) [inline forms-flow-bpm/sample.env:73]: same here

File: forms-flow-bpm/src/main/java/org/camunda/bpm/extension/commons/connector/HTTPServiceInvoker.java [hunk 3]
```diff
@@ -38,6 +40,8 @@ public class HTTPServiceInvoker {
     private ObjectMapper bpmObjectMapper;
     @Autowired
     private Properties integrationCredentialProperties;
+    @Value("${formsflow.ai.enableCustomSubmission}")
+    private boolean enableCustomSubmission;
```
Comment (shibin-aot) [inline forms-flow-bpm/src/main/java/org/camunda/bpm/extension/commons/connector/HTTPServiceInvoker.java:44]: use integrationCredentialProperties to access this

File: forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/BPMFormDataPipelineListener.java [hunk 4]
```diff
@@ -44,6 +47,8 @@ public class BPMFormDataPipelineListener extends BaseListener implements TaskLis
     private ObjectMapper bpmObjectMapper;
     @Autowired
     private HTTPServiceInvoker httpServiceInvoker;
+    @Value("${formsflow.ai.enableCustomSubmission}")
```
Comment (shibin-aot) [inline forms-flow-bpm/src/main/java/org/camunda/bpm/extension/hooks/listeners/BPMFormDataPipelineListener.java:50]: integrationCredentialProperties - can be used

File: forms-flow-bpm/src/main/resources/application.yaml [hunk 5]
```diff
@@ -17,6 +17,9 @@ server:
         max-age: 1800
 
 formsflow.ai:
+  enableCustomSubmission: ${CUSTOM_SUBMISSION_ENABLED}
```
Comment (shibin-aot) [inline forms-flow-bpm/src/main/resources/application.yaml:20]: you can create a subproperty called forms and place the new properties under it. Make more sense

**Review by shibin-aot — CHANGES_REQUESTED** [review]
I added the changes, also FormBpm and FormBpmFiltered and FormConnecter is pending

**Review by AbijithS-aot — APPROVED** [review]

**Comment by auslin-aot** [comment]
Updated in https://github.com/AOT-Technologies/forms-flow-ai/pull/669

**Pull request closed without merging**